client/src/components/Explore.js

```diff
@@ -59,41 +69,64 @@ function Explore() {
 				item
 				justify="center"
 				alignItems="center"
-				spacing={6}
+				spacing={5}
 				className={classes.cardContainer}
 			>
-				<Grid item xs={6} md={3}>
+				<Grid item xs={12} lg={3}>
```

Comment:
these exploreCard are the same style, they are just different location, country and name, you can create a list with each element {location:..., country:..., imgName:...} and map it. It helps code shorter

Reply:
I thought about that but decided against it since it was only going to be 8 static values and viewing the component with its props is easier than seeing it through an obj or list and using the map function. However, Id like to hear what others think too cause I could be totally wrong! :)

Reply:
You should definitely be generating these components by iterating through it, much cleaner and allows for easy extendability if we increase the # of locations.

Reply:
Will update this! Thanks Chloe for initially catching it @chloe-nguyen-12

Reply:
![mapExplore](https://user-images.githubusercontent.com/45303104/105092700-96a1fc00-5a6f-11eb-95c0-7f38795755b3.png)

Also added to new commit